sunpy/net/jsoc/jsoc.py

```diff
@@ -900,3 +900,10 @@ def _can_handle_query(cls, *query):
     @classmethod
     def _attrs_module(cls):
         return 'jsoc', 'sunpy.net.jsoc.attrs'
+
+    @classmethod
+    def register_values(cls):
```

Comment:
Do we want to put this method here at all if it's not implemented? I assume that it can just be left off the client completely?

Reply:
Fixed.